pkg/mcs/resource_manager/server/resource_group.go

```diff
@@ -145,17 +145,23 @@ func FromProtoResourceGroup(group *rmpb.ResourceGroup) *ResourceGroup {
 	switch group.GetMode() {
 	case rmpb.GroupMode_RUMode:
 		if settings := group.GetRUSettings(); settings != nil {
-			ruSettings = &RequestUnitSettings{
-				RU: NewGroupTokenBucket(settings.GetRU()),
+			ruSettings = &RequestUnitSettings{}
+			if settings.GetRU() != nil {
```

Comment:
what will happen if settings.GetRU() is nil?

Reply:
`RUSettings.RU` will be an empty GroupTokenBucket which has nil pointer of `Settings`.  If `Settings` is a nil pointer, GroupTokenBucket can not handle RequestRU